### ISSUE TITLE
Documentation:  Provide instructions on how to develop/update install scripts

### ISSUE THREAD

**Issue by drmikecrowe** [jpillora/installer#13]
I had a couple of issues hosting:

* I was able to deploy my own version and update the `.go` files w/o an issue
* When I tried to update the install script, I could not determine how to update `scripts/install.sh` and include. 
* I installed and was able run `statik -dest=.. -f -p=scripts -src=../scripts -include=*.sh` manually
* Could not determine how to deploy to my heroku instance.  Once I updated `scripts/statik.go`, I started getting 500 errors in heroku.

**Comment by jpillora** [comment]
Agreed, this can now be made much simpler with https://golang.org/pkg/embed/ newly released in go 1.16

Will update this issue, once we switch, then its just a `go build` and you're done

**Comment by jpillora** [comment]
Sorry, I'm also assuming we won't need the extra documentation if it works with a `go build` - but maybe it still needs docs...
